### PR TITLE
AOS Solicitor 2/5/Adultery

### PR DESCRIPTION
**Pull request by gavinclark**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DIV-5505

### Change description ###
Added field to handle 2 year separation, 5 year separation and adultery cases for respondent solicitor

Added to Draft and Update AOS event

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://div-cos-staging.service.core-compute-aat.internal",
+      "cosUrl": "http://div-cos-pr-528.service.core-compute-preview.internal",
```
Comment (alexwakeman) [inline package.json]: REVERT THIS

File: definitions/divorce/json/AuthorisationCaseField.json [hunk 2]
```diff
@@ -12697,6 +12697,167 @@
     "UserRole": "caseworker-divorce-solicitor",
     "CRUD": "R"
   },
+  {
+    "LiveFrom": "19/07/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseFieldID": "RefusalDecision",
```
Comment (alexwakeman) [inline definitions/divorce/json/AuthorisationCaseField.json]: Are we meant to include this here?

Comment (alexwakeman) [reply definitions/divorce/json/AuthorisationCaseField.json]: i.e. the refusal journey stuff

File: definitions/divorce/json/FixedLists.json [hunk 3]
```diff
@@ -717,19 +717,36 @@
...
+    "ID": "RespondentWillDefendEnum2",
+    "ListElementCode": "No",
+    "ListElement": "The respondent will let the divorce proceed",
+    "DisplayOrder": 1
+  },
+  {
+    "LiveFrom": "06/01/2019",
+    "ID": "RespondentWillDefendEnum2",
+    "ListElementCode": "Yes",
+    "ListElement": "The respondent intends to defend the divorce  ",
```
Comment (iichr) [inline definitions/divorce/json/FixedLists.json]: empty space at the end but can ignore; why replace the recently introduced RespondentWillDefendEnum2 further down

Comment (alexwakeman) [reply definitions/divorce/json/FixedLists.json]: It was introduced here before you guys used it actually, although yours was merged first... decided to keep our one as a result. Sorry!

**Comment by nsingamsetty** [comment]
This is tested by Elisha and signed off by business. See DIV-5505 for evidences and sign off details.. Merging this PR on behalf of Elisha.

**Pull request merged**